lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -114,11 +119,34 @@ class ReportUIFeatures {
     // There is only a sticky header when at least 2 categories are present.
     if (Object.keys(this.json.categories).length >= 2) {
       this._setupStickyHeaderElements();
-      this._document.addEventListener('scroll', this._updateStickyHeaderOnScroll);
+
+      let elToAddScrollListener = this._document;
```

Comment:
just curious, how do folks feel about about ternary `const` expressions compared to the `let`/`if` construct?

Reply:
yeah i kinda leaned on a C++ style here for some reason ..... i can change it.